main.go

```diff
@@ -423,7 +423,18 @@ func run() (err error) {
 	return
 }
 
+// Handle panic if the crash occurs during mounting.
+func handlePanic() {
```

Comment:
We can rename this method as it looks like so generic - handlePanicWhileMounting().